lib/mongo/server/connection_pool.rb

```diff
@@ -36,6 +36,10 @@ class ConnectionPool
       # @since 2.9.0
       DEFAULT_MIN_SIZE = 0.freeze
 
+      # The default maximum number of connections that can be connecting at
+      # any given time.
+      DEFAULT_MAX_CONNECTING = 2.freeze
```

Comment:
Freezing an immutable constant doesn't really do anything, does it? My understanding is that the primary purpose of freezing constants is to prevent them from being modified in place. Is there another reason for freezing numbers (which are already immutable)?

Reply:
Good point, thank you. Fixed.